.gitignore

```diff
@@ -1,4 +1,158 @@
-.idea/
```

Comment:
Not a good idea to put everything possible to ignore file. Looks very messy and redundant. Better idea is to add ignores on demand.